src/libs/H.Formatters.Inferno/PipeClientExtensions.cs

```diff
@@ -27,33 +27,35 @@ public static class PipeClientExtensions
         Action<Exception>? exceptionAction = null)
     {
         client = client ?? throw new ArgumentNullException(nameof(client));
-        client.Connected += async (o, args) =>
+        client.Connected += async (_, connArgs) =>
```

Comment:
```suggestion
        client.Connected += static async (_, args) =>
```
I don't really like abbreviated names like this, which only make sense for the sake of avoiding collisions. I think it's worth using static lambda here and in other places where there are name intersections. I use this extensively in new code. In situations where you need a sender, use the lambda's first argument.

Reply:
Fine by me! :)